src/GuardedExecutor.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.23;
+
+import {IERC7821} from "./interfaces/IERC7821.sol";
+import {MinimalDelegationStorage, MinimalDelegationStorageLib} from "./libraries/MinimalDelegationStorage.sol";
+import {Call} from "./libraries/CallLib.sol";
+import {LibBytes} from "solady/utils/LibBytes.sol";
+import {EnumerableSetLib} from "solady/utils/EnumerableSetLib.sol";
+import {IGuardedExecutor} from "./interfaces/IGuardedExecutor.sol";
+
+/// @title GuardedExecutor
+/// @author modified from https://github.com/ithacaxyz/account/blob/main/src/GuardedExecutor.sol
+abstract contract GuardedExecutor is IGuardedExecutor {
+    using EnumerableSetLib for EnumerableSetLib.Bytes32Set;
+
+    ////////////////////////////////////////////////////////////////////////
+    // Constants
+    ////////////////////////////////////////////////////////////////////////
+
+    /// @dev Represents any key hash.
+    bytes32 public constant ANY_KEYHASH = 0x3232323232323232323232323232323232323232323232323232323232323232;
+
+    /// @dev Represents any target address.
+    address public constant ANY_TARGET = 0x3232323232323232323232323232323232323232;
+
+    /// @dev Represents any function selector.
+    bytes4 public constant ANY_FN_SEL = 0x32323232;
```

Comment:
Is there a good use case for this? 

If integrating w Universal Router this does essentially nothing, and will be true for other contracts that are weiroll style

I'm sort of against the fn selector permissions, and i guess keeping permissioning as general as possible, or as easy to move to another module as possible